packages/components/src/components/ScalarMarkdown/ScalarMarkdown.vue

```diff
@@ -1,28 +1,29 @@
 <script setup lang="ts">
 import { htmlFromMarkdown } from '@scalar/code-highlight'
-import { cx } from '@scalar/use-hooks/useBindCx'
-import { computed } from 'vue'
-
-const props = withDefaults(
-  defineProps<{
-    value?: string
-    withImages?: boolean
-    transform?: (node: Record<string, any>) => Record<string, any>
-    transformType?: string
-    clamp?: string | boolean
-    class?: string
-    withAnchors?: boolean
-    anchorPrefix?: string
-  }>(),
-  {
-    withImages: false,
-    withAnchors: false,
-  },
-)
+import { useBindCx } from '@scalar/use-hooks/useBindCx'
+import { computed, useTemplateRef } from 'vue'
+
+import type { ScalarMarkdownProps } from './types'
+
+const {
+  value,
+  transform,
+  transformType,
+  withImages = false,
+  withAnchors = false,
+  anchorPrefix,
+} = defineProps<ScalarMarkdownProps>()
+
+const { cx } = useBindCx()
+defineOptions({ inheritAttrs: false })
+
+// Expose the element ref to the parent component
+const ref = useTemplateRef('ref')
```

Comment:
Gotta change this name. We should not be using the name of a core Vue API for anything.

Reply:
Not gonna lie I'm surprised the linter let me do this 🤔 I'll update